libraries/cmd/unit_generic.cpp

```diff
@@ -1701,16 +1704,19 @@ bool Unit::Explode(bool drawit, float timeit) {
         if (!sub) {
             un = _Universe->AccessCockpit()->GetParent();
             if (this->getUnitType() == Vega_UnitType::unit) {
-                if (rand() < RAND_MAX * game_options()->percent_shockwave && (!this->isSubUnit())) {
-                    static string shockani(game_options()->shockwave_animation);
```

Comment:
And here's another static instance that doesn't get accessed.
This time, it sets SetFaceCam.